### DISCUSSION TITLE
RTL

### DISCUSSION THREAD

**Discussion by Alkimay** [nick-keller/react-datasheet-grid#126] [Feature requests]
Hey is it possible you will add RTL support

**Comment by nick-keller** [comment]
I am not familiar with what that implies, could you please help by pointing out what would need to be done?

**Comment by Alkimay** [comment]
we write in Hebrew, and the sheet direction is rtl like in xslx we can set the direction. 
i want that the columns will start from the right to the left if its possible

**Comment by nick-keller** [comment]
To be honest this sounds like a lot of work that I cannot provide for now. If you would like to contribute please feel free, I'll gladly help you out.

**Reply by ibrahim-gabr** [reply]
we are using your awesome package but we’re currently in need of RTL support,
and I would love to contribute towards implementing this feature.
Could you please guide me on where to begin, or point me to any relevant documentation to get started?
@nick-keller
